services/web-app/package.json

```diff
@@ -13,14 +13,15 @@
     "build": "npm run clean && next build",
     "clean": "rimraf .carbon public/.carbon .next .dev",
     "clean:data": "rimraf public/data",
-    "dev": "next dev",
-    "dev:clean": "npm run clean && next dev",
-    "dev:secure": "sh ./scripts/create-local-certificates.sh && RUNNING_SECURELY=1 NODE_TLS_REJECT_UNAUTHORIZED=0 node secure-dev-server.js",
+    "dev": "concurrently \"npm run start-proxy\" \"next dev -p 3001\"",
+    "dev:clean": "npm run clean && npm run dev",
+    "dev:secure": "concurrently  \"sh ./scripts/create-local-certificates.sh && RUNNING_SECURELY=1 NODE_TLS_REJECT_UNAUTHORIZED=0 node secure-dev-server.js\" \"RUNNING_SECURELY=1 npm run start-proxy\"",
     "dev:secure:clean": "npm run clean && npm run dev:secure",
     "lint": "next lint",
     "service:build": "npm run build",
-    "start": "next start",
-    "test": "echo 'No tests defined yet!'"
+    "start": "concurrently \"npm run start-proxy\" \"next start -p 3001\"",
+    "test": "echo 'No tests defined yet!'",
+    "start-proxy": "node proxy-server.js"
```

Comment:
Let's sync up and re-think all of our scripts here to see if we can simplify them based on our common workflows and use cases.